spring-pulsar-spring-boot-autoconfigure/src/test/java/org/springframework/pulsar/autoconfigure/PulsarAutoConfigurationTests.java

```diff
@@ -210,7 +210,7 @@ void customProducerInterceptorsOrderedProperly() {
 	void consumerBatchPropertiesAreHonored() {
 		contextRunner
 				.withPropertyValues("spring.pulsar.listener.max-num-messages=10",
-						"spring.pulsar.listener.max-num-bytes=101", "spring.pulsar.listener.batch-timeout-millis=50")
+						"spring.pulsar.listener.max-num-bytes=101B", "spring.pulsar.listener.batch-timeout=50ms")
```

Comment:
Should we document these new formats in the doc?

Reply:
These should be self documenting in that they are config props and the doc is auto-generated.